NAS-102561: Finish draft rewrite of Pool Encryption docs (#1353)
freenas/freenas-docs: userguide/storage.rst

can result in irrevocably losing all access to the data stored in the
encrypted pool!**

.. _Encryption Operations:

Encryption Operations
^^^^^^^^^^^^^^^^^^^^^

Encryption operations are seen by clicking |pool-lock| for the encrypted
pool in
:menuselection:`Storage --> Pools`.
These options are available:

* :guilabel:`Lock`: Only appears after a passphrase is created. Locking
  a pool restricts data accessability in %brand% until the pool is
  unlocked.

  Selecting this action requires entering the passphrase. The pool
  status changes to :literal:`LOCKED`, :guilabel:`Pool Operations` are
  limited to *Export/Disconnect*, and |pool-lock| changes to
  |pool-unlock|.

* :guilabel:`Unlock`: Decrypt the pool by clicking |pool-unlock| and
  entering the passphrase *or* uploading the recovery key file. Only
  the passphrase is used when both a passphrase and a recovery key are
  entered. The services listed in :guilabel:`Restart Services` restart
  when the pool is unlocked. This enables %brand% to begin accessing
  the decrypted data. Individual services can be prevented from
  restarting by opening :guilabel:`Restart Services` and deselecting

>   them. However, this can prevent those services from properly accessing

Reviewer: Fix pause.

```suggestion
  them. Deselecting services can prevent them from properly accessing
```
